geopandas/array.py

```diff
@@ -1091,3 +1093,18 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return self._binop(other, operator.ne)
+
+    def __contains__(self, item):
+        """
+        Return for `item in self`.
+        """
+        if pd.api.types.is_scalar(item) and pd.core.dtypes.missing.isna(item):
```

Comment:
```suggestion
        if _isna(item):
```

We have a version above that specifically checks scalars (or otherwise use `pd.isna()`, since `pd.core.dtype.missing` is private pandas API)